src/api/add-profile-skill-mutation.ts

```diff
@@ -0,0 +1,38 @@
+import { useParams } from "react-router-dom";
+import { AddProfileSkillInput, Profile } from "cv-graphql";
+import { RouteParams } from "@/router";
+import i18n from "@/i18n";
+import { gql, useMutation } from "@apollo/client";
+import { useNotificationContext } from "@/helpers/notification";
+import { GET_USER_QUERY } from "@/modules/users/api/get-user-query.ts";
+
+export const ADD_PROFILE_SKILL = gql`
+    mutation addProfileSkill($skill: AddProfileSkillInput!) {
+        addProfileSkill(skill: $skill) {
+            id
+            skills {
+                name
+                category
+                mastery
+            }
+        }
+    }
+`;
+
+export const useAddProfileSkill = () => {
+    const { [RouteParams.UserId]: userId } = useParams();
+    const { showNotification } = useNotificationContext();
+
+    return useMutation<Profile | Promise<Profile>, { skill: AddProfileSkillInput }>(
+        ADD_PROFILE_SKILL,
+        {
+            refetchQueries: [{ query: GET_USER_QUERY, variables: { userId } }],
```

Comment:
Refetching wrong query here (since skills seem to be part of "profile" query, not "user")